src/r5py/util/classpath.py

```diff
@@ -3,7 +3,8 @@
 """Make sure R5 is in the class path, download it if not."""
 
 import hashlib
-import os.path
+import os
+import pathlib
```

Comment:
should we import this like `from pathlib import Path` ? It would make the code a bit cleaner, imo.